lib/paypal_nvp.rb

```diff
@@ -58,6 +69,9 @@ def call_paypal(data)
       http.verify_mode = OpenSSL::SSL::VERIFY_NONE
     end
 
+    http.open_timeout = @open_timeout
+    http.read_timeout = @read_timeout
```

Comment:
@jaxi What happens if `@open_timeout` or `@read_timeout` aren't set, i.e they are `nil`. Does `Net::HTTP` use the defaults if set as `nil`?